neo/public/javascripts/common/services.js

```diff
@@ -74,6 +74,18 @@ angular.module('cloudberry.common', [])
       ];
     }
 
+    function getPopulationTarget(parameters){
+      var targetLowerCase = parameters.geoLevel;
+      var targetUpperCase = targetLowerCase.charAt(0).toUpperCase() + targetLowerCase.substr(1);
+      return {
+        joinKey: [targetLowerCase],
+        dataset: "twitter.ds" + targetUpperCase + "Population",
```

Comment:
Do you mean to explicitly declare three separate query objects for state/county/city and use some conditions to map to them? And I wonder the reason to avoid using string concatenation. Is it because it's more error prone?